modules/spotxBidAdapter.js

```diff
@@ -85,7 +85,7 @@ function Spotx() {
 
       bid.cpm = KVP_Object.spotx_bid;
       bid.vastUrl = url;
-      bid.ad = url;
+      bid.ad = KVP_Object.spotx_ad_key;
```

Comment:
Does a key render?

Reply:
No a key will not render a video however the url that will be used to retrieve the ad is saved under the bid.vastUrl key. We were saving the same url to both bid.vastUrl and bid.ad. The reason I added the key to the bid.ad parameter is to be used with custom DFP targeting. SpotX uses custom DFP parameters for spotx_bid and spotx_ad_key to target ads in DFP which we now have access to both in the bid response.